style.css

```diff
@@ -0,0 +1,147 @@
+body {
+  margin: 0;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  color: #4d4d4d;
+  background: #ebebe0;
+}
+
+h1 {
+  text-align: center;
+  font-size: 50px;
+}
+
+h3 {
+  text-align: center;
+}
+
+/* Cards */
+.card-container {
+  margin: 50px auto;
+  width: 100%;
+  background: #669999;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+}
+
+article {
+  width: 300px;
+  height: 450px;
+  text-align: center;
+  margin: 3% 0;
+  padding-top: 30px;
+  border-radius: 16px;
+  border: 16px solid #e5e5e5;
+  box-shadow: 5px 5px 5px 0 #666;
+}
+
+p {
+  margin: 3px 16px;
+  font-weight: 300;
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+.titles {
+  font-size: 25px;
+}
+
+img {
+  height: 150px;
+  padding: 30px;
+}
+
+article:hover {
+  cursor: pointer;
+}
+
+.killer-deck {
+  background: #738c73;
+  background-image: radial-gradient(#85ad99 15%, transparent 16%),radial-gradient(#669966 15%, transparent 16%);
+  background-size: 35px 35px;
+  background-position: 0 0, 20px 20px;
+}
+
+.killer-text {
+  text-align: left;
+}
+
+.indent {
+  padding-left: 3px;
+}
+
+.weapon-deck {
+  background-color: #c2b2a3;
+  background-image: radial-gradient(#ede6de 15%, transparent 16%),radial-gradient(#cc9966 15%, transparent 16%);
+  background-size: 35px 35px;
+  background-position: 0 0, 20px 20px;
+}
+
+.room-deck {
+  background: #996666;
+  background-image: radial-gradient(#d6c2c2 15%, transparent 16%),radial-gradient(#c27070 15%, transparent 16%);
+  background-size: 35px 35px;
+  background-position: 0 0, 20px 20px;
+}
+
+/* Reveal */
+.reveal {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+h2 {
+  text-align: center;
+}
+
```

Comment:
Food for thought: Maybe you want to have an indicator that the button is clickable?
You could have a cursor as you have on the cards or maybe a small hover/transition?

Really nice that the button change color after it is clicked! 👍

Reply:
Ooh! Good call. I'll add some kind of hover effect when I come back around to this code.

Reply:
Just adding this bit into the CSS now.